src/tests/Home.test.js

```diff
@@ -0,0 +1,8 @@
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<Home />).toJSON();
+
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Please use React testing library to create some integration tests that simulate user interaction

Reply:
Thanks for your suggestion, although the Home page is not interactive at all, the user doesn't have anything to interact with.